packages/web-components/fast-foundation/src/utilities/whitespace-filter.ts

```diff
@@ -7,9 +7,9 @@
  * @public
  */
 export function whitespaceFilter(value: Node, index: number, array: Node[]) {
-    return (
-        value.nodeType === Node.TEXT_NODE &&
-        typeof value.nodeValue === "string" &&
-        !!value.nodeValue.trim().length
-    );
+    return value.nodeType !== Node.TEXT_NODE
+        ? true
+        : value.nodeType === Node.TEXT_NODE &&
```

Comment:
this check is now redundant and can be removed.

Reply:
Truth. Removed